src/lib/glee.ts

```diff
@@ -27,6 +27,12 @@ type ClusterAdapterRecord = {
 }
 
 export default class Glee extends EventEmitter {
+  static createMessage(value: any) {
+    throw new Error('Method not implemented.')
+  }
+  private _createMessage(value: any) {
+    throw new Error('Method not implemented.')
+  }
```

Comment:
This should not be here. You don't need to touch this file to add your adapter :)

```suggestion
```

Reply:
noo i am not touching these files should i remove this lines from 30 to 35?

Reply:
Yes, remove these lines. It's not needed.